tests/tests_pytorch/loggers/test_neptune.py

```diff
@@ -117,9 +117,6 @@ def test_neptune_pickling(self, neptune):
     def test_online_with_wrong_kwargs(self, neptune):
         """Tests combinations of kwargs together with `run` kwarg which makes some of other parameters unavailable
         in init."""
-        with self.assertRaises(ValueError):
```

Comment:
Not entirely sure if this test should be removed. 

I will have engineering double check it.

Reply:
@shnela please check here.

Reply:
Added back